tests/compute/half-texture.slang.expected

```diff
@@ -4,7 +4,7 @@ standard error = {
 standard output = {
```

Comment:
We should probably shift this test to use a `.slang.glsl` baseline instead of just having a `.expected` file of SPIR-V. Updating the baseline like this is not a *huge* problem, but it would be easy for us to fail to notice a semantic change to the output SPIR-V because we are unlikely to read every single line of it.